### PR TITLE
Submit Portfolio

### PR DESCRIPTION
**Pull request by kibazohb**
First Pull Request Made.

### CODE REVIEW

File: README.md
Comment (classm-sps) [inline README.md]: nit: the double quote at the end of the sentence looks extra and should probably be removed. (I use "nit:" to prefix nit picky comments that are purely cosmetic)

Comment (classm-sps) [inline README.md]: Consider adding the link to your deployed AppEngine site to make it easier for people to find it.

File: portfolio/src/main/webapp/WEB-INF/blog.html
Comment (classm-sps) [inline portfolio/src/main/webapp/WEB-INF/blog.html]: This file looks incomplete.

File: portfolio/src/main/webapp/WEB-INF/styles/reset.css
Comment (classm-sps) [inline portfolio/src/main/webapp/WEB-INF/styles/reset.css]: It's generally a good idea (for a lot of tools) to end files with a newline (but not a blank, empty line). The red "slashed circle" plus "carriage return" indicator on the diff is telling you there is no trailing newline. Please add one.

(A word about my review style: I'll often make comments -- particularly about general principles -- that apply to multiple places in the PR. I generally only make the comment once, with the hope that you'll go through and apply the comment/principle as appropriate throughout the PR. This is one of those comments :))

File: portfolio/src/main/webapp/index.html
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: The formatting here is quite nice, but please consider wrapping to 80 columns

File: portfolio/src/main/webapp/script.js
Comment (classm-sps) [inline portfolio/src/main/webapp/script.js]: Given that you've removed the reference to this file in index.html, probably should just remove this file entirely.

File: portfolio/src/main/webapp/style.css
Comment (classm-sps) [inline portfolio/src/main/webapp/style.css:1]: From a formatting standpoint, prefer to have a single space before opening curly braces. So:
```
body {
  ...
}
```

It's also consistent with the existing `#content` style :)

**Review by classm-sps — CHANGES_REQUESTED** [review]
Great work, Kiba! I've made a few mostly cosmetic comments that it would be great to clean up before merging to master. In addition to the comments on specific lines of code:

There's an unrelated file deletion from the `stanley/` directory. I don't think that should be included in this PR, please revert that deletion.

File: portfolio/src/main/webapp/WEB-INF/styles/reset.css
Comment (kibazohb) [reply portfolio/src/main/webapp/WEB-INF/styles/reset.css]: Okay, I'll take note to your style.

File: portfolio/src/main/webapp/index.html
Comment (kibazohb) [reply portfolio/src/main/webapp/index.html]: okay, another formatting issue. I'll take notes for sure.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (classm-sps) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: This is a week 2 change, and shouldn't be part of the week 1 PR. You have a couple of options:

1.  revert this change, fully resolve this week one PR, then make this change again
2. rebase this change onto a new branch

Given how simple this change is, option 1 is probably the fastest. Option 2 is a good thing to learn how to do. Your call.

**Review by classm-sps — CHANGES_REQUESTED** [review]
There are 3 open comments that I'd definitely like addressed before approving. I'd actually prefer that you also make the requested cosmetic changes, but I won't block you for them this time since we're already into week 2. Going forward I will probably be more insistent.

File: README.md
Comment (kibazohb) [reply README.md]: I am having trouble deploying the app to the server. Once I get it done, I will add the URL.

**Review by kibazohb — COMMENTED** [review]
Viewed all changes. I think you have permissions to merge it to master.

File: README.md
Comment (kibazohb) [reply README.md]: I deployed my app, should be ready to merge.

File: portfolio/src/main/webapp/index.html
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: This line is malformed. There is a missing `>` for the `<a` tag, with the result that the `button` tag is not being respected.

I think this should be either:

```
<a href="./blog.html">Blog!</a>
```

or

```
<button type="button" onclick="./blog.html">Blog!</button>
```

File: portfolio/src/main/webapp/styles/article-text.css
Comment (classm-sps) [inline portfolio/src/main/webapp/styles/article-text.css:4]: I'm not going to hold up the pull request over this, but please note for future work that styling for CSS is inconsistent across this PR. Sometimes there's a space before an opening `{`, sometimes there's not, sometimes there are several. Same thing for closing braces. Using a consistent style helps readers (reviewer, future maintainers) understand your code more easily -- as engineers, we're trained to notice patterns and deviations from them, so these inconsistencies stand out and distract from the intent of the code.

**Review by classm-sps — CHANGES_REQUESTED** [review]
Great job -- your portfolio looks really nice and adding the second page is a nice touch! Just one comment about the Blog! button, and then I'll approve this PR.

I'll freely admit that CSS is not my strong suit, so I've only given those changes a cursory review.

File: portfolio/src/main/webapp/styles/article-text.css
Comment (kibazohb) [reply portfolio/src/main/webapp/styles/article-text.css:4]: Yes, that does make sense, I'll make the code a lot more consistent with respect to a coding style.

File: portfolio/src/main/webapp/index.html
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: Looks like a funky merge happened here. (There are merge marker here on line 10, and also on line 19 and 36). Please fix up index.html so that it looks completely the way you want it to and make sure to test it!

Comment (kibazohb) [reply portfolio/src/main/webapp/index.html]: Yes I did not notice that. I fixed it. Thanks a lot for the help.

**Review by classm-sps — APPROVED** [review]
Thanks for your patience with this review...great job, and merge away!

**Pull request merged**
